frontend/src/services/actions.ts

```diff
@@ -60,130 +62,20 @@ const messageActions = {
   },
 };
 
-function getRiskText(risk: ActionSecurityRisk) {
-  switch (risk) {
-    case ActionSecurityRisk.LOW:
-      return "Low Risk";
-    case ActionSecurityRisk.MEDIUM:
-      return "Medium Risk";
-    case ActionSecurityRisk.HIGH:
-      return "High Risk";
-    case ActionSecurityRisk.UNKNOWN:
-    default:
-      return "Unknown Risk";
```

Comment:
I think the security analyzer uses this to render the risk of commands before they are confirmed by Invariant/the user

Reply:
Moved this logic into the chat-slice now